syz-fuzzer/proc.go

```diff
@@ -276,17 +276,17 @@ func (proc *Proc) enqueueCallTriage(p *prog.Prog, flags ProgTypes, callIndex int
 	})
 }
 
-func (proc *Proc) executeRaw(opts *ipc.ExecOpts, p *prog.Prog, stat Stat, enableDF Stat, disableDF Stat) *ipc.ProgInfo {
+func (proc *Proc) executeRaw(opts *ipc.ExecOpts, p *prog.Prog, stat Stat) *ipc.ProgInfo {
 	if opts.Flags&ipc.FlagDedupCover == 0 {
 		log.Fatalf("dedup cover is not enabled")
 	}
 	dfDisable := prog.DFetchAnalysis(p)
 	if dfDisable {
-		atomic.AddUint64(&proc.fuzzer.stats[disableDF], 1)
+		atomic.AddUint64(&proc.fuzzer.stats[StatEnableDF], 1)
 		opts.Flags |= (1 << 6)
```

Comment:
Please use ipc.FlagDoubleFetch instead of 1<<6

Reply:
isn't i need to do something like that opts.Flags &= (^ipc.FlagDoubleFetch), as if we need it by default every thing to be true, so we will need to set here 6th bit with zero to disable DoubleFetching